homeassistant/components/startca/sensor.py

```diff
@@ -81,32 +134,18 @@ def __init__(self, startcadata, sensor_type, name):
         """Initialize the sensor."""
         self.client_name = name
         self.type = sensor_type
-        self._name = SENSOR_TYPES[sensor_type][0]
-        self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
-        self._icon = SENSOR_TYPES[sensor_type][2]
+        metadata = SENSOR_TYPES[sensor_type]
```

Comment:
Pass in the entity_description into this constructor and set it directly like so:

```suggestion
        self.entity_description = entity_description
```